src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -166,11 +166,11 @@ internal JsonConfigFileAccessor()
         /// not interfere with PowerShell initialization
         /// </summary>
         /// <returns>Returns the directory if present or creatable. Throws otherwise.</returns>
-        private string GetAppDataConfigDirectory()
+        private string GetCurrentUserConfigDirectory()
         {
             if (null == appDataConfigDirectory)
```

Comment:
it seems `appDataConfigDirectory` is initialized in the constructor `JsonConfigFileAccessor()`. Then why do we need this method?